src/components/Table/TableHead/TableHead.jsx

```diff
@@ -279,7 +282,7 @@ const TableHead = ({
                     https://github.com/IBM/carbon-components-react/issues/1088 */}
             <Checkbox
               disabled={isDisabled}
-              id="select-all"
+              id={`select-all${selectAllId}`}
```

Comment:
What if we instead modified this component to take the tableId prop and pass it down from `<Table>`?

It would then be unique to the table but also would be configurable

Reply:
Sounds like a good solution, I'll get that done!